### PR TITLE
Feature/s3 init docs

### PR DESCRIPTION
**Pull request by simonhir**
**Description**

Init docs for integrations. In specific s3-integration.

**Reference**

Issues #50

### CODE REVIEW

File: refarch-integrations/refarch-s3-integration/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <groupId>de.muenchen.refarch</groupId>
+        <artifactId>refarch-s3-integration-java-starter</artifactId>
+        <version>...</version>
+    </dependency>
+</dependencies>
+```
+
+## Configuration
+
+Following are the properties to configure the different modules. Some of them a custom defined and others are synonyms
```
Comment (devtobi) [inline refarch-integrations/refarch-s3-integration/README.md:38]:
```suggestion
Following are the properties to configure the different modules. Some of them are custom defined and others are synonyms
```

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/configuration/S3IntegrationClientAutoConfiguration.java [hunk 2]
```diff
@@ -30,6 +33,7 @@
 )
 @RequiredArgsConstructor
 @EnableConfigurationProperties(S3IntegrationClientProperties.class)
+@PropertySource(value = ResourceUtils.CLASSPATH_URL_PREFIX + "application.yml", factory = YamlPropertySourceFactory.class)
```
Comment (devtobi) [inline refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/configuration/S3IntegrationClientAutoConfiguration.java:36]: Why is this line needed?

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/factory/YamlPropertySourceFactory.java
Comment (devtobi) [inline refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/factory/YamlPropertySourceFactory.java:1]: Why is this file needed?

File: refarch-integrations/refarch-s3-integration/README.md
Comment (devtobi) [inline refarch-integrations/refarch-s3-integration/README.md:1]: I would replace all references to `*.muenchen.de` with different example. (like mydomain.com) or something. I don't think we should include company specific Domains and subdomains in documentation.

**Review by devtobi — CHANGES_REQUESTED** [review]

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/configuration/S3IntegrationClientAutoConfiguration.java (hunk 2 above)
Comment (simonhir) [reply refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/configuration/S3IntegrationClientAutoConfiguration.java:36]: As far as i know only Properties in the application module itself are auto loaded.

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/factory/YamlPropertySourceFactory.java
Comment (simonhir) [reply refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/factory/YamlPropertySourceFactory.java:1]: See above

File: refarch-integrations/refarch-s3-integration/README.md
Comment (simonhir) [reply refarch-integrations/refarch-s3-integration/README.md:1]: Done

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/configuration/S3IntegrationClientAutoConfiguration.java (hunk 2 above)
Comment (devtobi) [reply refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/configuration/S3IntegrationClientAutoConfiguration.java:36]: Ok, looks fine to me.

**Review by devtobi — APPROVED** [review]
LGTM
